jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -1729,6 +1729,23 @@ ecma_object_get_class_name (ecma_object_t *obj_p) /**< object */
 
         switch (ext_obj_p->u.built_in.id)
         {
+#ifndef CONFIG_DISABLE_ARRAYBUFFER_BUILTIN
+          case ECMA_BUILTIN_ID_ARRAYBUFFER_PROTOTYPE:
+#endif /* !CONFIG_DISABLE_ARRAYBUFFER_BUILTIN */
+#ifndef CONFIG_DISABLE_TYPEDARRAY_BUILTIN
+          case ECMA_BUILTIN_ID_TYPEDARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_INT8ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_UINT8ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_INT16ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_UINT16ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_INT32ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_UINT32ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_FLOAT32ARRAY_PROTOTYPE:
+          case ECMA_BUILTIN_ID_UINT8CLAMPEDARRAY_PROTOTYPE:
+#if CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT64
+          case ECMA_BUILTIN_ID_FLOAT64ARRAY_PROTOTYPE:
+#endif /* CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT64 */
+#endif /* !CONFIG_DISABLE_TYPEDARRAY_BUILTIN */
```

Comment:
But I think it is useful to put assert here. for example, it helps us to catch the issue. If we extend other types in the future, and forget the implement their get_class_name, it will alert us.

I prefer to reserve the assert